src/bgfx.cpp

```diff
@@ -344,9 +344,13 @@ namespace bgfx
 		{
 			BGFX_FATAL(true
 				&& g_platformData.ndt     == _data.ndt
+#if !BX_PLATFORM_ANDROID
+				// Changing window handle on Android is actually possible:
+				&& g_platformData.nwh     == _data.nwh
+#endif
 				&& g_platformData.context == _data.context
 				, Fatal::UnableToInitialize
-				, "Only backbuffer pointer and native window handle can be changed after initialization!"
```

Comment:
`"Only backbuffer pointer (and native window handle on Android OS) can be changed after initialization!"`